framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -133,6 +138,8 @@ public ValueAnnotatedTypeFactory(BaseTypeChecker checker) {
         // this line just registers the alias. The BottomVal is never used.
         addAliasedAnnotation(MinLen.class, BOTTOMVAL);
 
+        addAliasedAnnotation(PolyAll.class, POLY);
```

Comment:
This is a bug, then. I removed this and the tests that rely on them being equivalent started failing (`Polymorphic2.java` is the failing test).